stresstest-aws/scripts/node/disk-usage.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+while read -a LIST; \
+do timestamp=$(date +%s%N); \
+        printf '%s\n' 'disk-usage,folder="'${LIST[1]}'" bytes='${LIST[0]}' '$timestamp; \
```

Comment:
```suggestion
        printf 'disk-usage,folder="%s" bytes=%s %s\n', ${LIST[1]}, ${LIST[0]}, $timestamp; \
```
Wouldn't that make more sense if using printf?